src/stylesheets/testimonial.scss

```diff
@@ -0,0 +1,40 @@
+
+.section-testimonial {
+    background: linear-gradient(to right, #ececec, #e2eceb)
+}
+
+.section-title {
+    text-align: center;
+    padding: 0rem 1rem;
+}
+
+.section {
+    padding: 7rem 0rem;
+    position: relative;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+    vertical-align: top;
+}
+
+.img-circle {
+    border-radius: 50%;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.margin-btm {
+    margin-bottom: 2rem;
+}
+
+.testimonial-item {
+    margin-bottom: 3rem;
```

Comment:
You can use an `scss` mixin for margin that takes value as argument

Reply:
I did that initially but I still would need both these classes. I will do that.